odyssey.conf

```diff
@@ -210,7 +210,7 @@ cache_coroutine 0
 #
 # 16KB by default.
 #
-coroutine_stack_size 4
+coroutine_stack_size 16
```

Comment:
I understand the reason, just to be clear that this will increase overall memory consumption per client